composer.json

```diff
@@ -10,11 +10,11 @@
         "ast"
     ],
     "require": {
-        "rector/rector": "~0.12",
+        "rector/rector": "~0.12.19",
```

Comment:
We're broken against 0.12.18 and it's `bootstrap.php` code which autoloads TestCase too early and Drupal cannot rewrite it.